backend/package.json

```diff
@@ -19,6 +19,7 @@
     "bcrypt": "5.0.1",
     "cloudinary": "1.25.0",
     "datauri": "3.0.0",
+    "dayjs": "1.10.4",
```

Comment:
You don't need it here and on the frontend part because you use it from the shared
Remove this package from frontend and backend